sql/core/src/test/scala/org/apache/spark/sql/execution/QueryExecutionSuite.scala

```diff
@@ -350,14 +350,19 @@ class QueryExecutionSuite extends SharedSparkSession {
   }
 
   test("SPARK-47764: Cleanup shuffle dependencies - RemoveShuffleFiles mode") {
```

Comment:
I just realized that the test has a jira number. Should i create a new test with the jira id used for this change? @dongjoon-hyun please advise

Reply:
IMO you can extend the existing tests when the new functionality can be easily covered by those. But why didn't you adjust all `SPARK-47764: ...` tests?

Reply:
@peter-toth I modified only `SPARK-47764: Cleanup shuffle dependencies - RemoveShuffleFiles mode`, since my change was specific to `org.apache.spark.sql.execution.RemoveShuffleFiles`.

Reply:
No, your PR affects other `ShuffleCleanupMode`s too so the other 2 should work with both AQE enabled and disabled as well.

Reply:
I have added blocks to other tests.
@peter-toth @dongjoon-hyun Can you take look please